modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -149,6 +149,24 @@ password: password
 
 .. At the top of the catalog package, find the *Mobile* category that indicates that the mobile features of {org-name} {product-name} are succesfully installed.
 
+== Shutting down and restoring your cluster
+
+To shut down the local cluster run
+
+[source,bash]
+----
+ oc cluster down
+----
+
+To bring the cluster back with the state in tact run the following commands replacing `<IP_ADDRESS>` with your clusters IP
```

Comment:
maybe primitive, but should we tell where users can obtain this?

Reply:
Is that covered earlier in the doc? It is the same IP as your local machine

Reply:
@maleck13  does below assume you're running commands from mobile-core repo? What if the release is distributed as a tarball?